### DISCUSSION TITLE
Drone parts of interest

### DISCUSSION THREAD

**Discussion by dannyjacobs** [dannyjacobs/ECHO#52] [General]
Following Larry's convincing case for continuing custom builds  (see discussion #50), heres a thread for dropping parts and bits which might update the current build using the now discontinued E800 thrust system (ESCs, motors, props).
## Current system
The Mark 7 design from 2018 (see memo #51) uses the DJI E800 kit. The E800 is a 350kV (slow for heavy lift) with 13" props. The ESCs supply up to 20A at 26V.  Together this system delivers thrust of 2100g/rotor. On a hex this amounts to 12.6kg of total lift.   The motor stators mount to 35x10mm patterns and each weigh 106g.   Plastic props are cheap and light.
The E800 props use a custom twist-on fitting which reduces mass, hardware, and dramatically simplifies setup cutting cost and increasing reliability. 

## New Options
It appears that DJI is probably going out of the kit business. The E800 kit is no longer sold with nothing equivalent offered to replace it. However, it is a well performing system and provides a useful real-world baseline to start from. 

 - [Tmotor 400kV](https://store.tmotor.com/goods.php?id=318), more specs at [robotshop](https://www.robotshop.com/en/tmotor-uav-brushless-motor-u5-kv400.html)
     - Good: similar speed. 30A max current (compare to E800 at 20A), available in ESC+Prop kit.
     - Bad: double the mass (169 vs E800 106), kit comes with carbon fiber props (safety and cost), CF props seem to need _two_ screws. Blah

**Comment by dannyjacobs** [comment]
Another Bad for the Tmotor kV400 kit is that the props come in either 14 or 15".  This will require a bigger frame.

**Comment by christopherbarbarie** [comment]
Following up from the meeting, another motor option from Tmotor is the [MN3510 KV360](https://store.tmotor.com/goods.php?id=337), they have it available on [Robotshop](https://www.robotshop.com/ca/en/t-motor-navigator-mn3510-360kv-brushless-motor.html?gclid=CjwKCAjw7cGUBhA9EiwArBAvotdifwdUext_P-HddgTnaRalJ2Pc3jZZPjxZeR6td9ZV8Ifk9IhDdBoCoWcQAvD_BwE) as well. It's around the same weight as the DJI motor in the E800 kit but it has the same issue/inconvenience of needing two screws for mounting the props. This would also work with a 30A max ESC and the prop size seems to fit the Tarot 690s frame according to Ecalc.

**Comment by christopherbarbarie** [comment]
The specs for the new pixhawk flight controllers got released, here are my initial thoughts: the pixhawk 6x runs on the same stm32h753 processor as the cube orange but has a higher clock speed (480Mhz vs 400Mhz) it also supposedly uses a new IMU noise reduction system which should improve stabilization, the pinout is also similar but the 6x has 3 telemetry ports which is one more than the cube and has an ethernet port. The 6c on the other hand is supposed to be the cost effective option, with a stm32h743 processor and a smaller form factor board. It uses the same "new noise reduction" IMU tech as the 6x. It seems like the 6x is currently listed at $335USD which compared to our suppliers for the cube orange is actually cheaper
